### PR TITLE
Add Contributor Code of Conduct

### PR DESCRIPTION
**Pull request by lizgene**
Because it's never too soon to add a code of conduct.

### CODE REVIEW

**Comment by lizhubertz** [comment]
@pamo Re: Your question on quotes (I deleted the other branch / had some weirdness cleaning up commits before I saw your note) - I prefer dumb `"quotes"` so am happy to stick to that and standardize whenever new study guides are brought in from other sources. 👍

File: README.md [hunk 1]
```diff
@@ -24,3 +24,9 @@ Start Jekyll
 bundle exec jekyll serve
 ```
 For more information about Jekyll development, check out the [usage documentation](https://jekyllrb.com/docs/usage/) on the platform's site.
+
+## Code of Conduct
+
+In order to have a more open and welcoming community, Radical Barracuda adheres to a [code of conduct]((docs/CONTRIBUTING.md)).
```
Comment (pamo) [inline README.md]: Wrong link

**Review by pamo — CHANGES_REQUESTED** [review]
We should have some contributing guidelines in the future, but for now, link to the conduct doc instead.

File: README.md (hunk 1 above)
Comment (lizgene) [reply README.md]: Thanks 👍

**Comment by lizgene** [comment]
@pamo Contribution guidelines next up! Do you know a repo with really good ones we can copy from?

**Review by pamo — APPROVED** [review]

**Comment by pamo** [comment]
Not for a static site/blog. Does DU have some on the site's repo? Should be in the .github/ directory for the repo. https://github.com/blog/1184-contributing-guidelines

**Pull request merged**